src/pages/buyers/[buyerid]/catalogs/[catalogid]/categories/index.tsx

```diff
@@ -35,7 +35,7 @@ const CategoriesList = (props) => {
 
   const initCategoriesData = useCallback(
     async (catalogid: string) => {
-      const categoriesList = await Categories.List<ICategory>(catalogid)
+      const categoriesList = await Categories.List<ICategory>(catalogid,{depth:"99"})
```

Comment:
Shouldn't the depth be "all"?

Reply:
The OrderCloud API does not allow circular references (1 parent ID per child), so technically @alphaklp limiting to 99 would not change the behavior of the user experience. Also, changing the query to depth=99 would equate to asking for all categories that have ParentID=X where X is the 99th child in a continuous string of parent-child relationships (i.e. parent-child-child-child-child... onto 99 levels deep). Which in my experience is highly unlikely for a real-world catalog structure. 

In general, category management UIs starts at the root (level=1 (or 0 I'm not 100% certain on that)) and as administrators expand each root level the list call adjusts to display the children of the subsequent level. Although, catalogs with under 1000 categories (10 API calls max), it might make sense to list all categories (regardless of level) and reproduce the parent child relationship on the client side.

Reply:
TL;DR:
For this specific issue, inputting depth=all is likely the correct query adjustment for the goal of displaying a full tree of categories (beyond the root).

Reply:
I updated to depth=all